packages/data/src/world/mm/stone_tower_temple.yml

```diff
@@ -72,7 +72,7 @@
 "Stone Tower Temple Center":
   dungeon: ST
   exits:
-    "Stone Tower Temple Water Room": "underwater_walking || short_hook_anywhere"
+    "Stone Tower Temple Water Room": "underwater_walking || short_hook_anywhere || can_dive_small"
```

Comment:
I feel like the Dexihand would be getting in your way, no? Should probably include methods to defeat it (does not need the soul)

Reply:
The dexihand is easily avoided with the silver scale.